modules/yieldmoBidAdapter.js

```diff
@@ -258,6 +258,7 @@ function addPlacement(request) {
     placementInfo.tid = transactionId;
   }
   if (request.auctionId) {
+    // TODO: fix auctionId leak: https://github.com/prebid/Prebid.js/issues/9781
```

Comment:
the fact that this is wrapped in a conditional indicates to me they are okay with nulls.